openshift/deploy.sh

```diff
@@ -34,6 +34,7 @@ is_set_or_fail OC_USERNAME "${OC_USERNAME}"
 #is_set_or_fail OC_PASSWD "${OC_PASSWD}"
 is_set_or_fail AWS_ACCESS_KEY_ID "${AWS_ACCESS_KEY_ID}"
 is_set_or_fail AWS_SECRET_ACCESS_KEY "${AWS_SECRET_ACCESS_KEY}"
+is_set_or_fail AWS_DEFAULT_REGION "${AWS_DEFAULT_REGION}"
```

Comment:
Shouldn't this be added to the env-template.sh if you are expecting it to be present in the environment?

Reply:
Can you please group it with other AWS parameters so that it makes easy reading?

Reply:
env-template had region but laying somewhere, re-arranged it as request into once place.